### PR TITLE
FlowsLoginAndMeny

### PR DESCRIPTION
**Pull request by tantting**
## Description

Provide a clear description of what the PR does and why its relevant

## Related Issue ( optional )

Link to any related issue

## Changes

Describe the changes in more detail, if necessary include explanation of why some choices were made.

## Screenshots & GIFs ( optional )

media to provide more visual context if necessary

## Notes & Comments

General notes and comments, such as if there is things to consider ahead of deployment

### CODE REVIEW

**Review by JosefForkman — CHANGES_REQUESTED** [review]
Good work!
move ExchangeRate and EnumsExchangeRate

File: RebelAllianceBank/Menu/CustomerMenu.cs [hunk 1]
```diff
@@ -4,14 +4,20 @@
...
 {
     public class CustomerMenu : Menu
     {
-        public CustomerMenu(IUser currentUser) : base(currentUser) { }
+        private Customer _currentCustomer;
+
+        public CustomerMenu(IUser currentUser) : base(currentUser)
+        {
+            _currentCustomer = (Customer?)CurrentUser; 
+        }
```
[the remark this answers is not in the document]
Comment (tantting) [reply RebelAllianceBank/Menu/CustomerMenu.cs:20]: I have added this in trello instead.

File: RebelAllianceBank/Menu/CustomerMenu.cs [hunk 2]
```diff
@@ -4,13 +4,19 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
```
[the remark this answers is not in the document]
Comment (tantting) [reply RebelAllianceBank/Menu/CustomerMenu.cs:6]: Fixed

**Review by JosefForkman — CHANGES_REQUESTED** [review]
:)

File: RebelAllianceBank/Menu/CustomerMenu.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (tantting) [reply RebelAllianceBank/Menu/CustomerMenu.cs]: sorry! Fixed :)

**Review by JosefForkman — APPROVED** [review]

**Pull request merged**